Octokit/Http/InMemoryCredentialStore.cs

```diff
@@ -15,7 +15,7 @@ public InMemoryCredentialStore(Credentials credentials)
 
         public Task<Credentials> GetCredentials()
         {
-            return Task.Factory.StartNew(() => _credentials);
+            return Task.FromResult(_credentials);
```

Comment:
This actually introduces some resource contention when parallelizing tests (but isn't the only one).

Do you have any objections to this @haacked?